### ISSUE TITLE
must set  query-log-enabled = true when want to log slow query only.

### ISSUE THREAD

**Issue by stanxing** [influxdata/influxdb#8881]
Bug report

Influxdb version = 1.3.5

I must set 'query-log-enabled = true' when I want to log slow query.

when I configure the influxdb.conf below:
influxdb.conf
[data]
 ...
  query-log-enabled = false

[coordinator]
  query-timeout = "60s"
  log-queries-after = "10ms"

The result is that influxdb didn't ouput log about any query. 
Then I set query-log-enabled = true, I can see all query log including 'slow query log'.
I think it is not reasonable. May be have any better workaround to  log slow query only?

**Comment by TriAnMan** [comment]
Guess that a doc notice for `log-queries-after` will be enough to workaround this issue in many cases.

**Comment by jimis** [comment]
@dgnorton could you maybe reopen the issue? Is there a way to log the slow queries, without logging all queries? If not, then I believe this is still a valid issue.

**Comment by dgnorton** [comment]
@jimis did you try setting `log-queries-after`?  https://docs.influxdata.com/influxdb/v1.7/troubleshooting/query_management/#log-queries-after

**Comment by jimis** [comment]
Yes of course, I have set it to `log-queries-after = 1s` together with `query-log-enabled = false`. It's not clear from the page you linked, if that combination works. I assumed it doesn't after finding this bug report.

**Comment by nilsgaeq** [comment]
This is apparently still an issue in version 1.9.3. Will there be a fix?
